core/settings.py

```diff
@@ -62,6 +72,18 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+if DEBUG_TOOLBAR_ENABLED:
+    INSTALLED_APPS += ["debug_toolbar"]
+    MIDDLEWARE += ["debug_toolbar.middleware.DebugToolbarMiddleware"]
+else:
+    if "debug_toolbar" in INSTALLED_APPS:
+        INSTALLED_APPS.remove("debug_toolbar")
+        MIDDLEWARE = [
+            mw
+            for mw in MIDDLEWARE
+            if "debug_toolbar.middleware.DebugToolbarMiddleware" not in mw
+        ]
```

Comment:
here we aren't checking if the debug_toolbar.middleware is in the middleware before removing it.
I guess it can be pretty safely assumed that it is.